src/private/EventMetricType.js

```diff
@@ -2,16 +2,16 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const glean = require('../glean');
-
 class EventMetricType {
-  constructor(category, name) {
+  constructor(glean, category, name) {
```

Comment:
I was kinda sad to pass the Glean instance around like we do on glean-core, that has caused my pain in glean-core development in the past. I think maybe since this is Javascript this maybe won't cause trouble.